### DISCUSSION TITLE
Welcome to Buzz Discussions!

### DISCUSSION THREAD

**Discussion by chidiwilliams** [chidiwilliams/buzz#373] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sircam19** [comment]
Hello I'm originally heavily interested in learning French but also taking a deep dive into transcription. Hello.
